test/e2e/run-all.js

```diff
@@ -63,21 +59,19 @@ async function main() {
     .strict()
     .help('help');
 
-  const { browser, debug, retries, snaps, mv3, nft } = argv;
+  const { browser, debug, retries, snaps, mv3 } = argv;
 
   let testPaths;
 
   if (snaps) {
     const testDir = path.join(__dirname, 'snaps');
     testPaths = await getTestPathsForTestDir(testDir);
-  } else if (nft) {
-    const testDir = path.join(__dirname, 'nft');
-    testPaths = await getTestPathsForTestDir(testDir);
   } else {
     const testDir = path.join(__dirname, 'tests');
     testPaths = [
       ...(await getTestPathsForTestDir(testDir)),
       ...(await getTestPathsForTestDir(path.join(__dirname, 'swaps'))),
+      ...(await getTestPathsForTestDir(path.join(__dirname, 'nft'))),
```

Comment:
The NFT tests should be run in the main e2e test run, alongside all the other specs